thorlcr/activities/choosesets/thchoosesetsslave.cpp

```diff
@@ -249,8 +249,9 @@ class InputCounter : public CSimpleInterface, implements IThorDataLink
     virtual void getMetaInfo(ThorDataLinkMetaInfo &info);
     virtual CActivityBase *queryFromActivity();
     virtual void dataLinkSerialize(MemoryBuffer &mb);
-    unsigned __int64 queryTotalCycles() const;
-
+    virtual unsigned __int64 queryTotalCycles() const;
+    virtual void debugRequest(MemoryBuffer &msg);
+    virtual void setInputSplitterOutputBase(CSplitterOutputBase *_input, unsigned _id=0){}
```

Comment:
trivial: missing space before {} (inconsistent with elsewhere)

But I'm not sure what this method is for.. The fact it is CSplitterOutputBase, but in all IThorDataLink definitions sounds wrong.